test/aries-js-worker/test/test.js

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Do we have open issues for more substantial tests?

Reply:
#1238 has been broken into sub tasks and that (big one) is one of those